app/models/follower.rb

```diff
@@ -0,0 +1,14 @@
+class Follower < ApplicationRecord
+    belongs_to :follower, class_name: 'Author'
+    belongs_to :followee, class_name: 'Author'
+    validates_associated :follower
+    validates_associated :followee
+    validates :follower_id, uniqueness: {scope: :followee_id}
+
+    scope :followers_count, ->(user_id){
+        where(follower_id: user_id).count
+    }
+    scope :following_count, ->(user_id){
+        where(followee_id: user_id).count
+    }
```

Comment:
You did it all the way around, followees_count should retrieve the count of user user_id is following, with your implementations your scope is retrieving the results where user_id is followed

```suggestion
    scope :followers_count, ->(user_id){
        where(followee_id: user_id).count
    }
```